test/test-server/fcpm_handlers.go

```diff
@@ -25,6 +25,7 @@ func handleFcpmComputePools(t *testing.T) http.HandlerFunc {
 					MaxCfu:      flinkv2.PtrInt32(1),
 					Region:      flinkv2.PtrString("us-west-1"),
 					Cloud:       flinkv2.PtrString("AWS"),
+					Environment: flinkv2.NewGlobalObjectReference("env-123", "-", "-"),
```

Comment:
Let's construct this with a struct definition and not a function? I'm not sure what the last two parameters are for by looking at this (although I get that you're trying to leave them out)

Reply:
Sure!